examples/azure-wordpress/porter.yaml

```diff
@@ -67,8 +72,8 @@ install:
         externalDatabase.database: "{{ bundle.parameters.database_name }}"
 
 uninstall:
-  # TODO: enable once the porter-azure mixin supports this action
-  # - azure:
+  # TODO: enable once the porter-arm mixin supports this action
```

Comment:
arm supports uninstall where the old azure mixin didn't. Maybe a follow-up would be to implement the stanza for uninstall

Reply:
Oh nice!  I'd be down to add it in this same PR, if you are?

Reply:
Oops, n/m.  Not quite ready for primetime: https://github.com/deislabs/porter-arm/blob/master/pkg/arm/uninstall.go

Reply:
Added a link to the issue around implementing uninstall (https://github.com/deislabs/porter-arm/issues/7) in the comment.

Reply:
🤦‍♀